semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtilsTest.java

```diff
@@ -193,6 +194,46 @@ public void testGetSqlURI() {
     when(e.getAttributes()).thenReturn(attributes);
     v = DbSemanticConventionUtils.getSqlURI(e);
     assertTrue(v.isEmpty());
+
+    attributes = SemanticConventionTestUtil.buildAttributes(
+        Map.of(
+            OTelDbSemanticConventions.DB_SYSTEM.getValue(),
+            SemanticConventionTestUtil.buildAttributeValue(OTelDbSemanticConventions.MYSQL_DB_SYSTEM_VALUE.getValue()),
+            OTelDbSemanticConventions.DB_CONNECTION_STRING.getValue(),
+            SemanticConventionTestUtil.buildAttributeValue("jdbc:mysql://mysql:3306/shop")));
+    when(e.getAttributes()).thenReturn(attributes);
+    v = DbSemanticConventionUtils.getSqlURI(e);
+    assertEquals("jdbc:mysql://mysql:3306/shop", v.get());
+  }
+
+  @Test
+  public void testGetSqlUrlForOtelFormat() {
```

Comment:
I wonder if we need some test where the user/password are included in the URL. Not sure tif that is a thing among jdbc.

Reply:
You mean if username/pwd is present we should drop the url?